index.html

```diff
@@ -101,7 +101,8 @@
       <p>
         This document defines a signal, transmitted over HTTP and through the DOM, that conveys a
         person's request to websites and services to not sell or share their personal information with
-        third parties. This standard is intended to work with existing and upcoming legal frameworks
+        third parties, or to have their data used for cross-organization ad targeting.
+        This standard is intended to work with existing and upcoming legal frameworks
```

Comment:
```suggestion
        third parties. This standard is intended to work with existing and upcoming legal frameworks
```

Reply:
Undoing this change. I think the implicit inclusion of ads stuff is best for the abstract.